recipe/meta.yaml

```diff
@@ -92,8 +92,8 @@ test:
 
 about:
   home: https://github.com/seekrcentral/seekr2_openmm_plugin.git
-  license: MIT
-  license_family: MIT
+  license: MIT & LGPL
+  license_family: MIT & LGPL
```

Comment:
So this is possible, but the syntax may need to change

SPDX supports [`OR`]( https://spdx.github.io/spdx-spec/v2.3/SPDX-license-expressions/#d42-disjunctive-or-operator ) as well as [`AND`]( https://spdx.github.io/spdx-spec/v2.3/SPDX-license-expressions/#d43-conjunctive-and-operator ). Depending on what applies would use one of these. If something else is intended entirely, there is more info in that doc about options

Also SPDX is aware that LGPL has versions. So would make sure to include the version information. Please see [this license list]( https://spdx.org/licenses/ ) for details

Lastly would recommend only using `license` (and dropping `license_family`). The latter is more limited. Plus we parse the SPDX and fill in `license_family` as part of the repodata. So only `license` should be needed

Happy to discuss any of that further. Hope that helps 🙂

Reply:
Ofc! Thanks for the follow up 🙏

Note for future readers, this is addressed in PR ( https://github.com/conda-forge/seekr2_openmm_plugin-feedstock/pull/14 )